src/VisualStudio/Core/Test/CodeModel/VisualBasic/RootCodeModelTests.vb

```diff
@@ -35,7 +35,7 @@ Module Goo
 End Module
 </code>
 
-            TestChildren(code, "MS", "My", "Microsoft", "System", "Goo", "Windows")
+            TestChildren(code, "MS", "My", "Microsoft", "System", "Goo", "FxResources", "Windows")
```

Comment:
?

Reply:
Previously, if `System.ValueTuple` types were not present, tuple types would not be error types. But now they are. So many (many!) IDE tests started failing. The solution was to add some references to the compilations created by `TestWorkspace.CreateCommonReferences`. Those references contain more stuff (namespaces), which is detected by this test.